packages/@aws-cdk/aws-events/lib/base-event-bus.ts

```diff
@@ -0,0 +1,130 @@
+import { Construct, IResource, Resource, Stack } from '@aws-cdk/core';
+import { CfnEventBus } from './events.generated';
+
+export interface IEventBus extends IResource {
+  /**
+   * The physical ID of this event bus resource
+   *
+   * @attribute
+   * @link https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-events-eventbus.html#cfn-events-eventbus-name
+   */
+  readonly eventBusName: string;
+
+  /**
+   * The ARN of this event bus resource
+   *
+   * @attribute
+   * @link https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-events-eventbus.html#Arn-fn::getatt
+   */
+  readonly eventBusArn: string;
+
+  /**
+   * The JSON policy of this event bus resource
+   *
+   * @attribute
+   * @link https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-events-eventbus.html#Policy-fn::getatt
+   */
+  readonly eventBusPolicy: string;
+
+  /**
+   * The partner event source to associate with this event bus resource
+   *
+   * @attribute
+   * @link https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-events-eventbus.html#cfn-events-eventbus-eventsourcename
+   */
+  readonly eventSourceName?: string;
+}
+
+export interface BaseEventBusProps {
+  /**
+   * The name of the event bus you are creating.
+   */
+  readonly eventBusName: string;
```

Comment:
Please make this able to be optional. Even if it's not optional in CFN, you can use `this.node.uniqueId` to build a fixed name in case the customer doesn't supply one.